### PR TITLE
docs(tutorial): Build a live app from the ground up

### PR DESCRIPTION
**Pull request by ComradeAERGO**

### CODE REVIEW

File: apps/docs/pages/examples/live-app-creation/manifest.mdx
Comment (RamyEB) [inline apps/docs/pages/examples/live-app-creation/manifest.mdx:1]: Already exist here : https://wallet.api.live.ledger.com/examples/test-live-app/import

Comment (RamyEB) [reply apps/docs/pages/examples/live-app-creation/manifest.mdx:1]: Redirecting or merging could be great

**Review by RamyEB — CHANGES_REQUESTED** [review]

File: apps/docs/pages/examples/live-app-creation/manifest.mdx
Comment (ComradeAERGO) [reply apps/docs/pages/examples/live-app-creation/manifest.mdx:1]: You're right!

**Review by Justkant — COMMENTED** [review]
I cannot see the pages on the documentation deployed for the branch https://wallet-api-git-doc-tutorial-ledgerhq.vercel.app/

File: apps/docs/pages/examples/live-app-creation/configuration.mdx
Comment (tjulien-ledger) [inline apps/docs/pages/examples/live-app-creation/configuration.mdx:1]: Similar to @RamyEB 's comment below, we have similar content here: https://wallet.api.live.ledger.com/react/simulator
simulator could be moved to examples under test your live app section instead

**Pull request merged**